src/pages/starter-kit/checkout/tax-usage.md

```diff
@@ -5,140 +5,105 @@ keywords:
   - App Builder
   - Extensibility
 ---
+
 # Tax API JavaScript usage
 
 To manage OOP tax integrations, Adobe Commerce provides a set of REST endpoints. The following sections describe how to use these endpoints in your JavaScript code.
 
 To call the Commerce REST endpoints, initialize the Adobe Commerce Client:
 
 ```javascript
-const { getAdobeCommerceClient } = require('../lib/adobe-commerce');
+const { getAdobeCommerceClient } = require("../lib/adobe-commerce");
 const commerceClient = await getAdobeCommerceClient(process.env);
 ```
 
 ## Create or modify a new OOPE tax integration
 
 `createOopeTaxIntegration` creates a new out-of-process tax integration:
 
-**Payload parameters:**
-
-| Parameter | Type   | Required | Description                               |
-| --------- | ------ | -------- | ----------------------------------------- |
-| `code`    | String | Yes      | Unique identifier for the tax integration. |
-| `title`   | String | Yes      | Display name of the tax integration.       |
-| `active`  | Boolean| No       | Status indicating if the tax integration is active. |
-| `stores`  | Array  | No       | List of store codes where the tax integration is available. |
+Check the [API reference](./tax-reference.md#create-or-modify-a-new-oope-tax-integration) for more details.
 
-<CodeBlock slots="heading, code" repeat="2" languages="javascript, json" />
+<CodeBlock slots="heading, code" repeat="1" languages="javascript" />
```

Comment:
I don't suggest using these code blocks when there is only one example.

Reply:
What is the correct code blocks for only one? I see other examples in src/pages/starter-kit/integration/code-samples/data-stock.md and src/pages/starter-kit/integration/code-samples/incoming-order.md but I don't know how to do it without codeblock.